### ISSUE TITLE
Cannot compile for iOS 16

### ISSUE THREAD

**Issue by fennelhans** [pichillilorenzo/flutter_inappwebview#1312]
- [x] I have read the [Get Started - Installation](https://inappwebview.dev/docs/get-started/installation/) section
- [x] I have read and done the [Get Started - Setup Android](https://inappwebview.dev/docs/get-started/setup-android/) section
- [x] I have read and done the [Get Started - Setup iOS](https://inappwebview.dev/docs/get-started/setup-ios/) section
- [x] I have already searched for the same problem

## Environment

| Technology           | Version       |
| -------------------- | ------------- |
| Flutter version      |  3.3.1        |
| Plugin version       |  5.4.3+7      |
| Android version      |  n/a          |
| iOS version          |  16           |
| Xcode version        |  14.0         |

Device information: 

## Description

Upgrading to macOS 12.6 + Xcode 14.0 + iOS 16 causes the plugin not to compile, throwing an error of `Stored properties cannot be marked potentially unavailable with '@available'`.

**Expected behavior:** 

The package should compile and run.

**Current behavior:**

The package prevents compilation from completing successfully, rendering my app unable to compile. 

## Steps to reproduce

1. Upgrade to Xcode 14
2. `flutter run`
3. Boom.

## Stacktrace/Logcat

```
Launching lib/main.dart on iPhone SE (3rd generation) in debug mode...
Xcode build done.                                           25.2s
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **
Xcode's output:
↳
    Writing result bundle at path:
    	/var/folders/n6/1mfx35y15fncn27hqswc_cnm0000gn/T/flutter_tools.tDV4hD/flutter_ios_build_temp_dirLe6e4G/temporary_xcresult_bundle
    /* com.apple.ibtool.document.warnings */
/Users/hans/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.4.3+7/ios/Storyboards/WebView.storyboard:global: warning: This file is set to build for a version older than the deployment target. Functionality may be limited. [9]
/Users/hans/.pub-cache/hosted/pub.dartlang.org/connectivity_plus-2.3.6+1/ios/Classes/PathMonitorConnectivityProvider.swift:31:5: warning: result of call to 'ensurePathMonitor()' is unused
        ensurePathMonitor()
        ^                ~~
/Users/hans/.pub-cache/hosted/pub.dartlang.org/connectivity_plus-2.3.6+1/ios/Classes/PathMonitorConnectivityProvider.swift:35:5: warning: result of call to 'ensurePathMonitor()' is unused
        ensurePathMonitor()
        ^                ~~
/Users/hans/.pub-cache/hosted/pub.dartlang.org/connectivity_plus-2.3.6+1/ios/Classes/ReachabilityConnectivityProvider.swift:23:5: warning: result of call to 'ensureReachability()' is unused
        ensureReachability()
        ^                 ~~
/Users/hans/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.4.3+7/ios/Classes/Types/UserScript.swift:13:6: error: stored properties cannot be marked potentially unavailable with '@available'
        @available(iOS 14.0, *)
         ^
    note: Building targets in dependency order
    warning: Run script build phase 'Run Script' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'Runner' from project 'Runner')
    warning: Run script build phase 'Thin Binary' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'Runner' from project 'Runner')
    Result bundle written to path:
    	/var/folders/n6/1mfx35y15fncn27hqswc_cnm0000gn/T/flutter_tools.tDV4hD/flutter_ios_build_temp_dirLe6e4G/temporary_xcresult_bundle
Swift Compiler Error (Xcode): Stored properties cannot be marked potentially unavailable with '@available'
/Users/hans/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-5.4.3+7/ios/Classes/Types/UserScript.swift:12:5

Could not build the application for the simulator.
Error launching application on iPhone SE (3rd generation).
Exited
```

**Comment by VictorUvarov** [comment]
The error goes away when I change `contentWorld` into a computed property. 

From:
```swift
@available(iOS 14.0, *)
lazy var contentWorld: WKContentWorld = WKContentWorld.page
```

to this:
```swift
private var _contentWorld: Any? = nil

@available(iOS 14.0, *)
var contentWorld: WKContentWorld {
    get {
        return (_contentWorld as? WKContentWorld) ?? WKContentWorld.page
    }
    set {
        _contentWorld = newValue
    }
}
```

**Comment by sanchan** [comment]
@VictorUvarov You are the hero we needed! Your solution works, my app now runs on iOS 16. Thank you!

**Comment by kaushikgodhani** [comment]
https://github.com/vocsyinfotech/flutter_inappwebview

you can use this one fork I have already update code and working in ios 16

**Comment by vbuberen** [comment]
Note, there is also a PR with fix, which is open for quite a long time already: https://github.com/pichillilorenzo/flutter_inappwebview/pull/1238

**Comment by pichillilorenzo** [comment]
Released new version `5.4.3+8` with the fix!
